ote/src/cljs/ote/ui/table.cljs

```diff
@@ -28,8 +28,8 @@
                }
               (when on-select
                 {:on-row-selection (fn [selected-rows]
-                                     (let [rows (vec rows)]
-                                       (on-select (map (partial nth rows) selected-rows))))})
+                                     (when (not (empty? selected-rows)) (let [rows (vec rows)]
```

Comment:
Use `seq` instead of `(not (empty? ...))` format the let to start on its own line